.circleci/config.yml

```diff
@@ -141,14 +144,17 @@ jobs:
           name: Install swiftlint
           command: brew install swiftlint
       - install-dependencies
+      - run:
+          name: Clear Derived Data
+          command: bundle exec fastlane run clear_derived_data
```

Comment:
surprised that this is needed, each job should start on a fresh CI machine

Reply:
Oh, this is not needed! I think I was just running into issue 😛

Reply:
Removed!